src/QirRuntime/test/FullstateSimulator/qir-test-simulator.ll

```diff
@@ -0,0 +1,1092 @@
+
+%Result = type opaque
+%Range = type { i64, i64, i64 }
+%Tuple = type opaque
+%Array = type opaque
+%Qubit = type opaque
+%Callable = type opaque
+
+@ResultZero = external global %Result*
+@ResultOne = external global %Result*
+@PauliI = constant i2 0
+@PauliX = constant i2 1
+@PauliY = constant i2 -1
+@PauliZ = constant i2 -2

```

Comment:
Funny :-)
Looks like the `i*` types (like `i64`, `i2`) are always _signed_, 
and the 2-bit unsigned value of `3` (having binary representation of `11`) 
is displayed as the signed 2-bit value of `-1` (also having binary representation of `11`).

And the same for 2-bit unsigned `2` displayed as 2-bit signed `-2` (both `10`).